app/src/main/java/com/spoony/spoony/core/designsystem/component/topappbar/TagTopAppBar.kt

```diff
@@ -0,0 +1,53 @@
+package com.spoony.spoony.core.designsystem.component.topappbar
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.core.designsystem.component.tag.LogoTag
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import com.spoony.spoony.core.designsystem.type.TagSize
+
+@Composable
+fun TagTopAppBar(
+    count: Int,
+    content: @Composable () -> Unit = {},
+    tagSize: TagSize = TagSize.Small,
+    showBackButton: Boolean = false,
+    onBackButtonClick: () -> Unit = {}
+) {
+    SpoonyBasicTopAppBar(
+        showBackButton = showBackButton,
+        onBackButtonClick = onBackButtonClick,
+        actions = {
+            LogoTag(
+                count = count,
+                tagSize = tagSize,
+                onClick = {},
```

Comment:
혹시나 해당 부분도 onClick 을 넘기는건 어떨까요?
```suggestion
fun TagTopAppBar(
    count: Int,
    content: @Composable () -> Unit = {},
    tagSize: TagSize = TagSize.Small,
    showBackButton: Boolean = false,
    onTagClick: () -> Unit = {},
    onBackButtonClick: () -> Unit = {}
) {
    SpoonyBasicTopAppBar(
        showBackButton = showBackButton,
        onBackButtonClick = onBackButtonClick,
        actions = {
            LogoTag(
                count = count,
                tagSize = tagSize,
                onClick = onTagClick,
```

Reply:
LogoTag는 클릭이 되지 않는 컴포넌트이기 때문에 onClick을 설정하지 않는게 맞다고 생각합니다! 현재 LogoTag에 onClick이 필수인자라 어쩔 수 없이 추가했는데 이 부분은 태그 머지 후 바로 수정하도록 하겠습니다!

Reply:
그리고 Pn룰 잊지 말고 추가해주세요😊

Reply:
넵 감사합니다~